cmd/query/app/static_handler.go

```diff
@@ -76,15 +76,11 @@ type loadedConfig struct {
 
 // NewStaticAssetsHandler returns a StaticAssetsHandler
 func NewStaticAssetsHandler(staticAssetsRoot string, options StaticAssetsHandlerOptions) (*StaticAssetsHandler, error) {
-	assetsFS := ui.StaticFiles
+	assetsFS := ui.GetStaticFiles(options.Logger)
 	if staticAssetsRoot != "" {
 		assetsFS = http.Dir(staticAssetsRoot)
 	}
 
-	if options.Logger == nil {
-		options.Logger = zap.NewNop()
-	}
```

Comment:
I see tests failing without this. Best is to fix the tests rather than defaulting logger here.